x-pack/plugins/cases/public/components/recent_cases/index.tsx

```diff
@@ -21,6 +21,7 @@ export interface RecentCasesProps extends Owner {
   allCasesNavigation: CasesNavigation;
   caseDetailsNavigation: CasesNavigation<CaseDetailsHrefSchema, 'configurable'>;
   createCaseNavigation: CasesNavigation;
+  hasWritePermissions: boolean;
```

Comment:
Used to determine if we should show a _create new case_ link in the recent cases overview section